specs/agents/tracing-instrumentation-db.md

```diff
@@ -188,6 +188,22 @@ If they don't, they MUST use a hard-coded list of URLs that correspond with the
 |`_.type`| `mongodb` | |
 |`_.name`| e.g. `customers` | Database name, same as `db.instance` if available  |
 
+#### `mongodb_capture_statement_commands` configuration
+
+This config var specifies the MongoDB command names for which the agent will capture the statement.
+Agents that support capturing MongoDB statements MUST implement this option.
+
+|                |  |
+|----------------|--|
+| Type           | `List<`[`WildcardMatcher`](../../tests/agents/json-specs/wildcard_matcher_tests.json)`>` |
+| Default        | `"find,aggregate,count,distinct,mapReduce"` |
+| Central config | `false` |
```

Comment:
Having just looked at @felixbarny's [PR](https://github.com/elastic/apm/pull/707), I was wondering what is the reason for making this option *not* centrally configurable?

Reply:
That's a good question, I picked non-central config for consistency with `elasticsearch_capture_body_urls` that has quite similar semantics which isn't part of Felix's PR.

As I was about to merge this PR I think it would be better to handle such change as a follow-up task, which would be in this case also make `elasticsearch_capture_body_urls` and `mongodb_capture_statement_commands` supported by central configuration.

Reply:
Agreed, a follow-up PR sounds like the way to go 👍 .

Reply:
I've just opened https://github.com/elastic/apm/issues/714 for 8.7 candidate.